tests/kallax.go

```diff
@@ -3120,29 +3162,32 @@ func (rs *PersonResultSet) Close() error {
 
 // NewPet returns a new instance of Pet.
 func NewPet(name string, kind string, owner *Person) (record *Pet) {
-	record = newPet(name, kind, owner)
-	if record != nil {
-		record.SetID(kallax.NewID())
-	}
-	return
+	return newPet(name, kind, owner)
```

Comment:
it can be solved just calling the `record.SetID()` in the record constructor (done in the generated `kallax.go`)
Anyway, that's what we were talking about, and we agreed on:
- user implements `newRecord()`
- then `kallax.go` will ensure that NewRecord() calls the `newRecord()`
Am I right?